### DISCUSSION TITLE
Changing logformat -from- json to something else like long

### DISCUSSION THREAD

**Discussion by darrenstalder** [renovatebot/renovate#16641] [Request Help] [answered]
I'm trying to get the debug logs to -not- be in json format but I can't find documentation for the logformat anywhere.  I learned that the logger that renovate uses is called bunyan, so I tried using 'long' as the format for my debug logs.  But neither RENOVATE_LOG_FORMAT=long nor RENOVATE_LOG_FILE_FORMAT=long seems to work, I still get json logs.

Any ideas on how to do this?  

Don't think that it matters but I'm using a self-hosting Renovate on a self-hosted GitLab on an Amazon Linux 2 EC2.

**Comment by rarkins** [comment, marked as answer]
By default you should see a "pretty" output for logs. You should only see json if you have set LOG_FORMAT=json. Unset that variable if so

**Reply by darrenstalder** [reply]
But I'm not.  We have:
RENOVATE_LOG_FILE: renovate-log.ndjson
RENOVATE_LOG_FILE_LEVEL: debug
set in our .gitlab-ci.yml file
So, if I put RENOVATE_LOG_FILE_FORMAT: "" in my .gitlab-ci.yml, that should work?
or would it just be RENOVATE_LOG_FORMAT and that will control the format of both onscreen and file?

It's just a pain to have to download the file and run it through jq to view it.

**Reply by rarkins** [reply]
The log *file* is always JSON, while stdout can be configured to be human or machine readable
